.pre-commit-config.yaml

```diff
@@ -134,12 +134,31 @@ repos:
           - --ignore=DL3008
   - repo: local
     hooks:
-      - id: prettier
-        name: prettier
+      - id: prettier-css
+        name: prettier-css
         language: system
-        entry: sh -c 'make frontend/node_modules && ./frontend/node_modules/.bin/prettier --write "$@"' --
+        entry: sh -c 'make frontend/node_modules && ./frontend/node_modules/.bin/prettier --write --tab-width 2 --print-width 79 --parser css "$@"' --
         require_serial: true
-        types_or: [javascript, jsx, ts, css, html, markdown]
+        files: '\.css$'
+      - id: prettier-jinja
+        name: prettier-jinja
+        language: system
+        entry: sh -c 'make frontend/node_modules && ./frontend/node_modules/.bin/prettier --write --tab-width 2 --print-width 79 --parser jinja-template --plugin "$(npm root -g)/prettier-plugin-jinja-template/lib/index.js" "$@"' --
+        require_serial: true
+        files: '\.(j2|jinja)$'
```

Comment:
This change isn't as straight forward to backport as I'd hoped. Prettier seems to just be constantly confused by the fact that `templates` is outside of `frontend`, and then can't find its plugins anymore. I suppose this is the reason for the `npm root -g` call here (which, for the record, I don't want to have in my codebase :) ).

The formatting by itself is IMO a good thing, and worth pursuing further. But considering that the entire build system will change in #81, I'm now at a point where I'd cut the losses, just accept the isolated template reformatting changes, and deal with automating the formatter calls after #81 lands.

So, let's just revert the changes to the pre-commit config and go ahead with merging this.